src/plugins/notebook/components/Notebook.vue

```diff
@@ -618,48 +635,20 @@ export default {
             const defaultNotebookObject = await this.getDefaultNotebookObject();
             if (!defaultNotebookObject) {
                 setDefaultNotebook(this.openmct, notebookStorage, this.domainObject);
-            } else if (objectUtils.makeKeyString(defaultNotebookObject.identifier) !== objectUtils.makeKeyString(notebookStorage.notebookMeta.identifier)) {
+            } else if (objectUtils.makeKeyString(defaultNotebookObject.identifier) !== objectUtils.makeKeyString(notebookStorage.identifier)) {
                 this.removeDefaultClass(defaultNotebookObject);
                 setDefaultNotebook(this.openmct, notebookStorage, this.domainObject);
```

Comment:
`setDefaultNotebook(this.openmct, notebookStorage, this.domainObject);` could be moved outside the if/else block since it's being called in both places.

Reply:
hmm, actually cant, need to `setDefaultNotebook` only on those 2 cases. I did flatten the logic